active/ListViewGridView/listviewgridview-designupdates.md

```diff
@@ -0,0 +1,622 @@
+
+Gets or sets the brush used to render the disabled selection checkmark.
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.PointerOverBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectedBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectedPressedBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectedDisabledBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPointerOverBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPressedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedPointerOverBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedPressedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxSelectedDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPointerOverBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxPressedBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxDisabledBorderBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckPressedBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckDisabledBrushProperty{ get; };` 
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectedInnerBorderBrushProperty{ get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.SelectionIndicatorCornerRadiusProperty { get; };`
+
+`static Windows.UI.Xaml.DependencyProperty ListViewItemPresenter.CheckBoxCornerRadiusProperty{ get; };`
+
+
+## New Visual State Group and States:
+
+### For reference, existing visual group and states for the SelectionMode==Multiple option: 
+
+**VisualStateGroup MultiSelectStates**
+
+VisualState MultiSelectDisabled  for selectionMode != Multiple
+
+VisualState MultiSelectEnabled   for selectionMode == Multiple
+
+
+### New visual group and states for the Selection==Single or Extended options: 
+
+**VisualStateGroup SelectionIndicatorStates**
+
+VisualState SelectionIndicatorDisabled  for selectionMode == None or Multiple
+
+VisualState SelectionIndicatorEnabled   for selectionMode == Single or Extended
+
+## Changes to existing APIs:
```

Comment:
I don't think these need to be described in DMC (are they today)? So this could go into the Appendix.